recipes/bytewax-feedstock/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "bytewax" %}
+{% set version = "0.16.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bytewax/bytewax/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 993fe70f3a206adc52aae1099c00e14a92ec55387dda4b0cda2b6f6fbf0ed35b
+  patches:
+    - ssl.patch
+
+build:
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - git
```

Why do you need git in build?

```suggestion
    - git
```